examples/shp/dot_product_benchmark.cpp

```diff
@@ -160,26 +197,31 @@ int main(int argc, char **argv) {
     double duration = std::chrono::duration<double>(end - begin).count();
     durations.push_back(duration);
 
-    if (v != v_serial) {
-      fmt::print("{} != {}\n", v, v_serial);
-    }
-    // assert(v == v_serial);
     sum += v;
   }
 
-  fmt::print("oneMKL executing on one device:\n");
-  fmt::print("Durations: {}\n", durations | rng::views::transform([](auto &&x) {
-                                  return x * 1000;
-                                }));
+  // fmt::print("oneMKL executing on one device: ");
+  // fmt::print("Durations: {}\n", durations | rng::views::transform([](auto
+  // &&x) {
+  //                                 return x * 1000;
+  //                               }));
 
   std::sort(durations.begin(), durations.end());
 
-  median_duration = durations[durations.size() / 2];
+  mkl_median = durations[durations.size() / 2] * 1000;
 
-  fmt::print("Median duration: {} ms\n", median_duration * 1000);
+  // fmt::print("Result: {}, Median duration: {} ms\n", sum, mkl_median);
 
-  fmt::print("Result: {}\n", sum);
 #endif
 
+  //  fmt::print("----------------------------------------------------------\n");
+  //  fmt::print("devices, size, seq_time, shp_time, dpl_time, mkl_time\n");
+  //  fmt::print("{}, {}, {}, {}, {}, {}\n", devices.size(), size, seq_median,
+  //  shp_median, dpl_median, mkl_median);
+
+  // fmt::print("----------------------------------------------------------\n");
+  // fmt::print("devices, size, shp_time, dpl_time\n");
```

Comment:
The code that is no longer used should be removed, especially if it is trivial (like printing messages).